jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gce-gci.yaml

```diff
@@ -109,6 +110,43 @@
                 export PROJECT="e2e-gce-gci-ci-serial"
                 export KUBE_MASTER_OS_DISTRIBUTION="gci"
                 export KUBE_NODE_OS_DISTRIBUTION="gci"
+        - 'release-1.4':  # kubernetes-e2e-gce-gci-ci-release-1.4
+            description: 'Runs all non-slow, non-serial, non-flaky, tests on GCE with GCI images in parallel on the release-1.3 branch.'
+            timeout: 50
+            trigger-job: 'kubernetes-build-1.4'
+            job-env: |
+                export JENKINS_PUBLISHED_VERSION="ci/latest-1.4"
+                export JENKINS_GCI_IMAGE_FAMILY="gci-53"
```

Comment:
@adityakali was mentioning `55`. We can go with `54` I think. I assume we can update the milestone separately from this PR?